teps/0025-hermekton.md

```diff
@@ -80,8 +82,14 @@ Tekton Pipelines will add support for a new "ExecutionMode" field on several obj
 That type will look like:
 
 ```go
+type HermeticMode string
+
+const (
+    NetworkingOff   HermeticMode = "networking-off"
+)
+
 type ExecutionMode struct {
-	Hermetic bool
+	Hermetic HermeticMode
```

Comment:
Is it worth adding `json:"hermetic,omitempty"` here while we're at it?

Reply:
sure, added :)